.github/workflows/main.yaml

```diff
@@ -0,0 +1,77 @@
+# Compare: https://github.com/pangeo-forge/pangeo-forge-recipes/blob/master/.github/workflows/main.yaml
+
+name: Tests
+
+on:
+  push:
+    branches: "*"
+    paths-ignore:
+    - 'docs/**'
+  pull_request:
+    branches: master
+    paths-ignore:
+    - 'docs/**'
+
+env:
+  PYTEST_ADDOPTS: "--color=yes"
+
+jobs:
+  test:
+    name: ${{ matrix.python-version }}-build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8, 3.9]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+      # - name: Cache conda
+      #  uses: actions/cache@v1
+      #  env:
+          # Increase this value to reset cache if ci/py${{ matrix.python-version }}.yml has not changed
+      #    CACHE_NUMBER: 0
+      #  with:
+      #    path: ~/conda_pkgs_dir
+      #    key: ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('ci/py${{ matrix.python-version }}.yml') }}
+      - name: setup miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: pfo-poetry
+      #    environment-file: ci/py${{ matrix.python-version }}.yml
+          python-version: ${{ matrix.python-version }}
+          auto-activate-base: false
+      #    use-only-tar-bz2: true
+      - name: install pangeo-forge-orchestrator plus deps
+        shell: bash -l {0}
+        run: |
+          conda install -c conda-forge poetry
+          poetry install
```

Comment:
Just wanted to echo this point: It's most prudent to use setuptools and python standards, even if there are better package management solutions out there. Package management and distribution is definitely a weak point of python. It's likely best to use the longest lived technology since poetry isn't established yet as the new standard (my perspective is in line with Ryans: this tool is popular in the web-development corner of python, but not data science). 

In my understanding, [these commands](https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html#summary) are the most appropriate from here on out to manage python project lifecycles.

Reply:
Complete! Packaging now matches `pangeo-forge-recipes`. (I will update the docs all at once in a later commit.)